ui/src/react-admin/modules/shared/components/PaginationBar/PaginationBar.tsx

```diff
@@ -0,0 +1,97 @@
+import { Button, Pagination } from '@meemoo/react-components';
+import clsx from 'clsx';
+import React, { FC } from 'react';
+
+import { Icon } from '../Icon';
+import { PaginationProgress } from '../PaginationProgress';
+
+import styles from './PaginationBar.module.scss';
+import { PaginationBarProps } from './PaginationBar.types';
+import { useTranslation } from '~modules/shared/hooks/useTranslation';
+
+const PaginationBar: FC<PaginationBarProps> = ({
+	className,
+	count,
+	onPageChange,
+	showBackToTop,
+	start,
+	total,
+}) => {
+	const { tHtml } = useTranslation();
+
+	const pageCount = Math.ceil(total / count);
+	const currentPage = start / count;
```

Comment:
I'd wrap these in a useMemo hook.

Reply:
I used the PaginationBar from `react-components`. This component will be deleted.